internal/pkg/describe/service.go

```diff
@@ -58,6 +58,7 @@ type configurations []*ServiceConfig
 
 func (c configurations) humanString(w io.Writer) {
 	fmt.Fprintf(w, "  %s\t%s\t%s\t%s\t%s\n", "Environment", "Tasks", "CPU (vCPU)", "Memory (MiB)", "Port")
+	fmt.Fprintf(w, "  %s\t%s\t%s\t%s\t%s\n", "-----------", "-----", "----------", "------------", "----")
```

Comment:
Maybe it would be better to generate those "-" (strings.Repeat) by code? In case we update the heading and we don't need to count  how many "-" we have.

Reply:
Yes! Thanks.